react-native-app/package.json

```diff
@@ -39,19 +39,21 @@
     "react-native": "^0.64.2",
     "react-native-encrypted-storage": "^4.0.2",
     "react-native-input-spinner": "^1.7.12",
-    "react-native-picker-select": "^8.0.4",
+    "react-native-material-dropdown": "^0.11.1",
     "react-native-modal": "^12.1.0",
     "react-native-permissions": "^3.2.0",
-    "react-native-material-dropdown": "^0.11.1",
+    "react-native-picker-select": "^8.0.4",
     "react-native-popup-menu": "^0.15.12",
     "react-native-safe-area-context": "^3.3.2",
     "react-native-screens": "^3.9.0",
     "react-native-share": "^7.3.4",
     "react-native-sound": "^0.11.1",
     "react-native-svg": "^12.1.1",
     "react-native-vector-icons": "^9.0.0",
+    "react-validation": "^3.0.7",
```

Comment:
I can't seem to find anywhere that uses these libraries in your changes - it looks like you built a custom solution.

If we don't use the packages then we should remove them.